pop.c

```diff
@@ -83,7 +83,7 @@ static int pop_read_header(struct PopData *pop_data, struct Header *h)
 {
   FILE *f = NULL;
   int rc, index;
-  long length;
+  size_t length;
```

Comment:
If we're using `size_t` here, we probably want to use `%zd` in the `sscanf()` below.

Reply:
Good catch. I changed it to `%zu` since `size_t` is inherently unsigned.